lighthouse-core/lib/stack-packs-test/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
this won't live in lighthouse for real but just added it as a hack to get it going

Reply:
sg! thx for making this